js/runner.go

```diff
@@ -344,7 +345,8 @@ func (r *Runner) HandleSummary(ctx context.Context, summary *lib.Summary) (map[s
 	}()
 	*vu.Context = ctx
 
-	handleSummaryWrapperRaw, err := vu.Runtime.RunString(summaryWrapperLambdaCode)
+	wrapper := strings.Replace(summaryWrapperLambdaCode, "/*JSLIB_SUMMARY_CODE*/", jslibSummaryCode, 1)
```

Comment:
I'm probably missing it, but what's the reason for having a wrapper and doing this replace? Since both `summaryWrapperLambdaCode` and `jslibSummaryCode` are static, couldn't we just bundle a single JS file?

Reply:
We probably could, but this insulates us from future changes in the jslib summary code, and it also keeps the scope a bit cleaner :man_shrugging:

Reply:
Ah, sorry, misread your question. I don't want to bundle a single JS file since I want it to be easy to update the `summary.js` when the one in jslib is updated